### PR TITLE
Added User Table and modified DatabaseContext

### PR DESCRIPTION
**Pull request by Heckin-Doggo**
- User table is created with fields listed in #21 
- Database context is set to make ID the key, User and Email are unique

### CODE REVIEW

**Comment by jpbream** [comment]
I just committed a change to this, so be sure to `git pull` before you make any more commits

File: Tech@HubAPI/Tech@HubAPITest/UserTest.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            user.Salt.Should().Equal(new byte[] { 6, 5, 4, 3, 2, 1 });
+            user.Email.Should().Be("test@email.com");
+            user.FirstName.Should().Be("Joe");
+            user.LastName.Should().Be("Mama");
+            user.Age.Should().Be(18);
+            user.Description.Should().Be("test");
+            user.ProfilePicturePath.Should().Be("usr/pics/joe.png");
+
+        }
+
```
Comment (jpbream) [inline Tech@HubAPI/Tech@HubAPITest/UserTest.cs:40]: Lets add a test for creating two users with duplicate emails/usernames, and ensure the database rejects that

**Review by jpbream — APPROVED** [review]
